docs/discounts/discounts_api.md

```diff
@@ -30,38 +30,79 @@ Discounts are applied in two places, listed in the [`DiscountType`](/api/php_api
 Regardless of activation place, discounts always apply to products and reduce their base price.
 
 To define when a discount activates and how the price is reduced, use rules and conditions.
-They make use of the [Symfony Expression language]([[= symfony_doc=]]//components/expression_language.html).
-Use the expression values provided below when using data migrations or when parsing REST API responses.
+They use the [Symfony Expression language]([[= symfony_doc=]]/components/expression_language.html) to express their logic.
 
 ### Rules
 
-Discount rules define how the calculate the price reduction.
-The following discount rule types are available:
+Discount rules define how to calculate the price reduction.
+The following discount rule types are available in the `\Ibexa\Discounts\Value\DiscountRule` namespace:
 
-| Rule type | Identifier | Description | Expression value |
-|---|---|---|---|
-| `Ibexa\Discounts\Value\DiscountRule\FixedAmount` | <nobr>`fixed_amount`</nobr> | Deducts the specified amount, for example <nobr>10 EUR</nobr>, from the base price | <nobr>`discount_amount`</nobr> |
-| `Ibexa\Discounts\Value\DiscountRule\Percentage` | <nobr>`percentage`</nobr> | Deducts the specified percentage, for example -10%, from the base price | <nobr>`discount_percentage`</nobr> |
+| Rule type<br>(identifier) | Description | Required expression value |
+|---|---|---|
+| `FixedAmount`<br><nobr>(`fixed_amount`)</nobr> | Deducts the specified amount, for example <nobr>10 EUR</nobr>, from the base price | <nobr>`discount_amount`</nobr> |
+| `Percentage`<br><nobr>(`percentage`)</nobr> | Deducts the specified percentage, for example -10%, from the base price | <nobr>`discount_percentage`</nobr> |
 
 Only a single discount can be applied to a given product, and a discount can only have a single rule.
 
+When creating a rule through other means than the user interface, you must pass the required expression values for the rule to be valid:
```

Comment:
It doesn't sound like simple English, "through other means" might be misunderstood.
```suggestion
When creating a rule, not with the user interface but an API, you must pass the required expression values for the rule to be valid:
```